velox/connectors/hive/SplitReader.cpp

```diff
@@ -141,6 +141,81 @@ void SplitReader::configureReaderOptions(
 void SplitReader::prepareSplit(
     std::shared_ptr<common::MetadataFilter> metadataFilter,
     dwio::common::RuntimeStatistics& runtimeStats) {
+  createReader();
+
+  if (testEmptySplit(runtimeStats)) {
```

Comment:
Is it good or reliable to do empty check based on runtime stats?

s/testEmptySplit/checkIfEmptySplit/

Reply:
@xiaoxmeng The check is not BASED on runtimeStats, but it needs to update it if the split can be skipped.
Regarding the name, would `isEmptySplit` or `hasEmptySplit` or `splitIsEmpty` be better? These names won't cause confusion whether returning true is empty or not empty.